src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/Editor_OutOfProc.cs

```diff
@@ -61,6 +61,12 @@ public bool IsCompletionActive()
             return _inProc.IsCompletionActive();
         }
 
+        public bool IsSignatureHelpActive()
+        {
+            WaitForSignatureHelp();
+            return _inProc.IsSignatureHelpActive();
```

Comment:
It might be worth commenting that the inproc method doesn't wait, before somebody gets the idea of trying to call it directly.